src/alpaca_eval/models_configs/ultralm-13b/configs.yaml

```diff
@@ -0,0 +1,14 @@
+ultralm-13b:
+  prompt_template: "ultralm-13b/prompt.txt"
+  fn_completions: "huggingface_local_completions"
+  completions_kwargs:
+    model_name: "/ultralm-13b" # local path
+    model_kwargs:
+      torch_dtype: 'float16'
+    max_new_tokens: 2048
+    temperature: 0.5
```

Comment:
is there a particular reason for temp to be 0.5 here? We've kept temp=0.7 for all the other models as default to maintain consistency, so we'd prefer to keep that if possible.